frontend/src/Editor/WidgetManager/widgetConfig.js

```diff
@@ -1142,20 +1142,8 @@ export const widgets = [
           schema: { type: 'string' },
         },
       },
-      enableTime: {
-        type: 'code',
-        displayName: 'Enable time selection?',
-        validation: {
-          schema: { type: 'boolean' },
-        },
-      },
-      enableDate: {
-        type: 'code',
-        displayName: 'Enable date selection?',
-        validation: {
-          schema: { type: 'boolean' },
-        },
-      },
+      enableTime: { type: 'toggle', displayName: 'Enable time selection?' },
```

Comment:
```suggestion
            enableTime: {
        type: 'toggle',
        displayName: 'Enable time selection?',
        validation: {
          schema: { type: 'boolean' },
          defaultValue: false,
        },
      },
      enableDate: {
        type: 'toggle',
        displayName: 'Enable date selection?',
        validation: {
          schema: { type: 'boolean' },
          defaultValue: true,
        },
      },
```

Reply:
Please can you add the validations for the widget property?

Reply:
Vaidations added